.rubocop.yml

```diff
@@ -115,5 +115,9 @@ RSpec/SpecFilePathFormat:
   Exclude:
     - 'spec/data_migrations/*'
 
+RSpec/AnyInstance:
+  Exclude:
+    - 'spec/**/*'
```

Comment:
Would it be better to continue allowing this on a case-by-case basis with an inline comment as the code being stubbed here is going to disappear fairly soon?